packages/teleterm/src/ui/DocumentCluster/ClusterResources/Databases/useDatabaseUsers.ts

```diff
@@ -0,0 +1,23 @@
+import useAsync from 'shared/hooks/useAsync';
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+
+export function useDatabaseUsers(dbUri: string) {
```

Comment:
IMO we don't need this file anymore if we just pass a promise to `MenuLoginAsync`.